retail/solutions/README.md

```diff
@@ -1,14 +1,11 @@
 # Retail Solutions
 
 ---
-This folder contains solutions for retail built using Microsoft Cloud (Azure, D365, O365, M365, Power Platform).
+This directory contains solutions for retail built using Microsoft Cloud (Azure, D365, O365, M365, Power Platform).
 
-Goal of solutions is to provide end-to-end reference implementation to address key retail-specific scenarios.
+Goal of retail solution is to provide considerations; recommendations and end-to-end reference implementation.
```

Comment:
"Goal of retail solutions are to provide considerations; recommendations and end-to-end reference implementation." just sound bettter I thi nk